Common/sim/Bloch.m

```diff
@@ -42,10 +42,13 @@
 end
 
 W = pi*Param.G.*omega2;
+R2f = 1/Param.T2f;
```

Comment:
@IanGagnon7 @tanguyduval Second thing I noticed is that 13 out of 22 of my unit test failed (Bloch_Test.m and BlochSol_Test.m), and is (as of now, at least partly) because of these additions (calls to Param.T2f and Param.F, which didn't exist previously).

I see that in BlochSol.m you handled this a different way for T2f ( you did an if isfield(T2f)... else), but didn't do it here or for F. 

It's easy for me to modify the unit tests to handle this addition, however maybe we want to resolve if we do want the "isfield(T2f)... else"in Bloch.m and BlochSol.m for both T2f and F (if so we should add those changes), otherwise maybe we should remove it in BlochSol for consistency.

Reply:
I added "if isfield(F)" and "if isfield(T2f)" in commit 2949ca5b4b6ab3dc45fb37b4e226e42ebedbdca3

Really handy the testing functions

Reply:
Great! And I agree - it makes debugging & refactoring much more painless!